src/main/java/it/gov/pagopa/initiative/statistics/exception/RestResponseExceptionHandler.java

```diff
@@ -14,14 +14,7 @@
 @ControllerAdvice
 public class RestResponseExceptionHandler {
 
-    // API
-    @ExceptionHandler({InitiativeStatException.class})
-    public ResponseEntity<ErrorDTO> handleInitiativeException(InitiativeStatException ex) {
-        return new ResponseEntity<>(new ErrorDTO(ex.getCode(), ex.getMessage()),
-                ex.getHttpStatus());
-    }
-
-    @ExceptionHandler({MethodArgumentNotValidException.class})
+    @ExceptionHandler(MethodArgumentNotValidException.class)
```

Comment:
this represent a common behavior that we could be interested to bring also in other applications!

Let's move it inside the ErrorManager itself ;)